healthcheck

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 HOST PORT
+
+Healthcheck for the RT docker container."
+    exit 1
+fi
+
+host="$1"
+port="$2"
+
+resp=$(curl -s -d @- -H "Host: rt.ocf.berkeley.edu" \
```

Comment:
Maybe add `-f`? Though presumably the `grep`s will fail below anyways.

Reply:
Done